pages/[slug]/[[...suffix]].js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
Could we rename this to `pages/[...path].js`? Would need small adaption in profile page but IMO worth it to also have this at the root level of the page folder.